dev-docs/bidders/appnexus.md

```diff
@@ -89,3 +90,35 @@ AppNexus supports using prebid within a mobile app's webview. If you are interes
 | `id`              | The App ID.                                                                                                                     | `'B1O2W3M4AN.com.prebid.webview'`                                        | `string`         |
 | `device_id`       | Object that contains the advertising identifiers of the user (`idfa`, `aaid`, `md5udid`, `sha1udid`, or `windowsadid`).         | `{ aaid: "38400000-8cf0-11bd-b23e-10b96e40000d" }`                       | `object`         |
 | `geo`             | Object that contains the latitude (`lat`) and longitude (`lng`) of the user.                                                    | `{ lat: 40.0964439, lng: -75.3009142 }`                                  | `object`         |
+
+<a name="custom-targeting-keys" />
+
+#### Custom Targeting keys
+
+Appnexus returns custom keys `buyerMemberId, dealPriority, dealCode` which can be additionally sent to adserver using bidderSettings. You can use following snippet to add these key value pairs.
```

Comment:
Suggest rewording: 

AppNexus returns custom keys that can be sent to the adserver through `bidderSettings`: `buyerMemberId`, `dealPriority`, and `dealCode`. The following snippet demonstrates how to add these custom keys as key-value pairs.